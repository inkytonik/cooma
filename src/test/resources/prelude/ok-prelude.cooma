{
    type Foo = Int
    def add(l : Foo, r : Foo) Foo = prim IntAdd(l, r)
    val sub = fun(l : Int, r : Int) prim IntSub(l, r)
    val x = {
        a = add(1, 2),
        b = sub(2, 1),
        c = "Hello",
        d = <<e = 3>>
    }
    {}
}
