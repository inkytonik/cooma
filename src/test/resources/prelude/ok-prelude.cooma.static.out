Foo :
  Type =
  Int;
add :
  (l : Int, r : Int) Int;
sub :
  (l : Int, r : Int) Int;
x :
  {
    a : Int,
    b : Int,
    c : String,
    d : <<
      e : Int
    >>
  };
